New Post - CSS `:is()` and `:where()` (#5475)
GoogleChrome/web.dev: src/site/content/en/blog/css-is-and-where/index.md

look at how they're different.

### The difference between `:is()` and `:where()`

When it comes to specificity, `:is()` and `:where()` strongly diverge. For a
brush up on specificity, see the [specificity module on Learn
CSS](https://web.dev/learn/css/specificity/).

In short
- `:where()` has no specificity.<br>`:where()` squashes all the specificity in
  the selector list passed as functional parameters. This is a first of its
  kind selector feature. 
- `:is()` takes the specificity of it's most specific
  selector.<br>`:is(a,div,#id)` has a specificity score of an ID, 100 points

Taking on the highest specificity selector from the list has only been a gotcha
for me when I was getting too excited about grouping. I was always able to
improve legibility by moving the high specificity selector to it's own selector
where it wouldn't have so much impact. Here's an example of what I mean:

```css
article > :is(header, #nav) {
  background: white;
}

/* better as */
article > header,
article > #nav {
  background: white;
}
```

With `:where()`, I'm waiting to see libraries offer versions with no
specificity. The specificity competition between author styles and library

> styles could come to an end. There would be no specificity to compete with as an
> author. CSS has been working on a grouping feature like this for quite some

Reviewer: ```suggestion
styles could come to an end. There would be no specificity to compete with when writing CSS. 
CSS has been working on a grouping feature like this for quite some
```
